src/hooks/context/contextService.spec.ts

```diff
@@ -0,0 +1,32 @@
+import { ContextService } from './ContextService';
+import * as os from 'os';
+import * as path from 'path';
+import * as fs from 'fs';
+import { ContextFile } from './models';
+
+describe('Context Service should', () => {
+	beforeEach(() => {
+		const contextFilePath = path.resolve(os.homedir(), '.asyncapi');
```

Comment:
`.asyncapi` string must be in some separate file, so we specify it just once and can reuse in main code and tests. Something like https://github.com/asyncapi/asyncapi-react/blob/master/library/src/constants.ts. Makes sense?

actually `path.resolve(os.homedir(), '.asyncapi');` could also be wrappend in a reusable util